client/injection/apiextensions/reconciler/apiextensions/v1/customresourcedefinition/reconciler.go

```diff
@@ -338,14 +338,14 @@ func (r *reconcilerImpl) updateStatus(ctx context.Context, logger *zap.SugaredLo
 // updateFinalizersFiltered will update the Finalizers of the resource.
 // TODO: this method could be generic and sync all finalizers. For now it only
 // updates defaultFinalizerName or its override.
-func (r *reconcilerImpl) updateFinalizersFiltered(ctx context.Context, resource *v1.CustomResourceDefinition, desiredFinalizers sets.String) (*v1.CustomResourceDefinition, error) {
+func (r *reconcilerImpl) updateFinalizersFiltered(ctx context.Context, resource *v1.CustomResourceDefinition, desiredFinalizers sets.Set[string]) (*v1.CustomResourceDefinition, error) {
```

Comment:
these are generated files, you need to [update the generator](https://github.com/knative/pkg/tree/main/codegen/cmd/injection-gen), and rerun them with `hack/update-codegen.sh`

Reply:
Yes, My bad.